docs/data-sources/certificate_authority.md

```diff
@@ -0,0 +1,69 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Generic"
+layout: "aci"
+page_title: "ACI: aci_certificate_authority"
+sidebar_current: "docs-aci-data-source-aci_certificate_authority"
+description: |-
+  Data source for Certificate Authority
+---
+
+# aci_certificate_authority #
+
+Data source for Certificate Authority
+
+## API Information ##
+
+* Class: [pkiTP](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/pkiTP/overview)
+
+* Supported in ACI versions: 1.0(1e) and later.
+
+* Distinguished Name Formats:
+  - `uni/tn-{name}/certstore/tp-{name}`
+  - `uni/userext/pkiext/tp-{name}`
+
+## GUI Information ##
+
+* Location: `Generic`
+
+## Example Usage ##
+
+```hcl
+
+data "aci_certificate_authority" "example" {
+  name = "test_name"
+}
+
+data "aci_certificate_authority" "example_tenant" {
+  parent_dn = aci_tenant.example.id
+  name      = "test_name"
+}
+
+```
+
+## Schema ##
+
+### Required ###
+
+* `name` (name) - (string) The name of the Certificate Authority object.
+
+### Optional ###
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_tenant](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/fvTenant) ([fvTenant](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvTenant/overview))
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the Certificate Authority object.
+* `annotation` (annotation) - (string) The annotation of the Certificate Authority object.
+* `cert_chain` (certChain) - (string) The PEM-encoded chain of trust from the trustpoint to a trusted root authority.
```

Comment:
should we rename this attribute certificate_chain?

Reply:
ok